Analysis/PhotonEnergyCorrect/include/photonCorrectionProcessor.h

```diff
@@ -85,6 +86,38 @@ class photonCorrectionProcessor : public marlin::Processor {
   float _endcap_gaus2_sigm{};
 
 
+  float _phiBias_barrel_p0_1  {};
```

Comment:
This list of parameters could be summarized by a vector. It would also simplify and shorten the configuration files. Could you turn this long series of parameters to 3 vectors?